types/jsreport-core/jsreport-core-tests.ts

```diff
@@ -1,17 +1,24 @@
 import JsReport = require('jsreport-core');
 
+const template: JsReport.Template = {
+    content: '<h1>Hello {{:foo}}</h1>',
+    engine: 'jsrender',
+    recipe: 'phantom-pdf'
+};
+
+const options: JsReport.RequestOptions = { };
+
+const request: JsReport.Request = {
```

Comment:
`const request = {` will be OK.
I think this may be the reason you add `?` to all the properties of `Request`.

Reply:
agreed you could go with an untyped object and cast it as the type but I feel like it's more intuitive to be able to work with both.

```
const request = { ... } as JsReport.Request;
// and
const request2: JsReport.Request = { ... };
```